migration/migrator_validate.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	zktrie "github.com/kroma-network/zktrie/types"
```

Comment:
Could you move this line to the end of the imports?

Reply:
Maybe my formatter settings are different from yours. I'll check what the problem is.